### ISSUE TITLE
Hyperparameters of IPPO

### ISSUE THREAD

**Issue by FisherguanbaoYu** [marlbenchmark/on-policy#9]
Are the choices of IPPO hyperparameters the same as MAPPO shown in Table 12? The only difference is the value of "use_centralized_V" (False for IPPO, True for MAPPO), right? Thanks!

**Comment by zoeyuchao** [comment]
It is true.

**Comment by FisherguanbaoYu** [comment]
OK. Then it's a bit strange that the performance on some maps (e.g., MMM) are not as good as shown in your paper without changing anything of the code. For 'MMM' map, I ran the following code:
```shell
#!/bin/sh
env="StarCraft2"
map="MMM"
algo="ippo"
exp="rnn"
seed_max=5

echo "env is ${env}, map is ${map}, algo is ${algo}, exp is ${exp}, max seed is ${seed_max}"
for seed in `seq ${seed_max}`;
do
    echo "seed is ${seed}:"
    python train/train_smac.py --env_name ${env} --algorithm_name ${algo} --experiment_name ${exp} --map_name ${map} --seed ${seed} --n_training_threads 127 --n_rollout_threads 8 --num_mini_batch 1 --episode_length 400 --num_env_steps 5000000 --ppo_epoch 15 --use_eval --use_wandb --use_centralized_V --use_recurrent_policy
done
```

**Comment by zoeyuchao** [comment]
#!/bin/sh
env="StarCraft2"
map="MMM"
algo="rmappo"
exp="ippo_test"
seed_max=5

echo "env is ${env}, map is ${map}, algo is ${algo}, exp is ${exp}, max seed is ${seed_max}"
for seed in `seq ${seed_max}`;
do
    echo "seed is ${seed}:"
    python train/train_smac.py --env_name ${env} --algorithm_name ${algo} --experiment_name ${exp} --map_name ${map} --seed ${seed} --n_training_threads 127 --n_rollout_threads 8 --num_mini_batch 1 --episode_length 400 --num_env_steps 5000000 --ppo_epoch 15 --use_eval --use_wandb --use_centralized_V
done

**Comment by zoeyuchao** [comment]
i have updated the code with some changes in config,py, so please execute "git pull" before you use the above parameters.

**Comment by FisherguanbaoYu** [comment]
So, you suggest to **Always use PopArt value normalization** in the paper, but actually the results are obtained using **valuenorm** (value of "--use_valuenorm" is true by default in config.py) instead of **Popart**. Also, are the values of "--use_value_active_masks" & "--use_policy_active_masks" always true across all maps? Thanks!

**Comment by zoeyuchao** [comment]
"--use_policy_active_masks" will always be true, "--use_value_active_masks" is not, it depends on the task or env. As for the popart issue, we just update the arxiv verison to describe more specifically.
